frontera/contrib/backends/sqlalchemy/__init__.py

```diff
@@ -132,7 +132,7 @@ def strategy_worker(cls, manager):
         if drop_all_tables:
             if model.__table__.name in inspector.get_table_names():
                 model.__table__.drop(bind=b.engine)
```

Comment:
Yeah, I think the check above ``if model.__table__.name in inspector.get_table_names()`` should do the same thing, but ``checkfirst=True`` looks better.